web/src/modules/administration/modules/injest-data/components/DataIngestionEditor.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+  <v-dialog
+    v-model="internalVisible"
+    persistent
+    max-width="700"
+  >
+    <v-card>
+      <v-toolbar
+        color="primary"
+        variant="dark"
+        title="Edit Location"
+      >
+        <v-spacer></v-spacer>
+        <v-btn
+          icon
+          @click="handleClose"
+          color="white"
+        >
+          <v-icon>mdi-close</v-icon>
+        </v-btn>
+      </v-toolbar>
+
+      <v-card-text>
+        <v-select
```

Comment:
Prefer wrapping all "input" type components in a `v-form` and using a "submit" type button to trigger submission. This allows you to make use of standard validation logic.

See this pattern https://github.com/icefoganalytics/wrap/blob/9da5e31931351ce911d441556caa388d4dc2ac35/web/src/components/positions/PositionEditCardForm.vue#L10 
You can consider the HeaderActionsFormCard as a v-form for the explanation here, as you probably don't need a HeaderActionsFormCard component.